lib/ts/recipe/emailpassword/components/emailPasswordAuth.tsx

```diff
@@ -109,4 +115,11 @@ class EmailPasswordAuth extends PureComponent<FeatureBaseProps, EmailPasswordAut
     };
 }
 
-export default EmailPasswordAuth;
+export default (function(): ReactComponentClass {
+    try {
```

Comment:
There are several cases of `try { ... } catch() { ... }`. This pattern in general indicates a lack of clear understanding of how the code in the try catch works. Whilst it's not a big problem, it's generally not the best coding practice.

Reply:
We need try catch to wrap `react-router-dom` optional dependency here.